Cargo.toml

```diff
@@ -46,6 +46,8 @@ services-google = [
   "dep:serde_json",
   "dep:jsonwebtoken",
   "dep:tokio",
+  "reqwest?/json",
+  "reqwest?/default-tls",
```

Comment:
`reqsign` fails to run only on itself because TLS is disabled due to `default-options = false`.

Reply:
You can enable tls in `dev-dependences`